### PR TITLE
Rename togglePatternType to setPatternType, get rid of toggled prop in RegexpToggle

### PR DESCRIPTION
**Pull request by attfarhan**
https://sourcegraph.slack.com/archives/CMT39K56Z/p1576524263013400

### CODE REVIEW

File: web/src/search/input/QueryInput.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ import { eventLogger } from '../../tracking/eventLogger'
 import { scrollIntoView } from '../../util'
 import { Suggestion, SuggestionItem, SuggestionTypes, createSuggestion, fuzzySearchFilters } from './Suggestion'
 import RegexpToggle from './RegexpToggle'
-import { SearchPatternType } from '../../../../shared/src/graphql/schema'
+// import { SearchPatternType } from '../../../../shared/src/graphql/schema'
```
Comment (rvantonder) [inline web/src/search/input/QueryInput.tsx]: is this comment a leftover?

Comment (attfarhan) [reply web/src/search/input/QueryInput.tsx]: Oh yeah this should be removed

**Comment by rvantonder** [comment]
I get a merge conflict when I try to merge this into my branch. I think we should just add my small number of relevant changes into `fa/accurate-toggle` and PR this against master.

Looks like the tests are updated so I assume this is fine for existing behavior. I can probably take care of writing a test for explicitly testing `structural implies not toggled`.

**Comment by rvantonder** [comment]
> I get a merge conflict when I try to merge this into my branch.

I don't think this is even needed, since I'm trying this locally and it seems to work fine without my changes?

**Comment by attfarhan** [comment]
> I get a merge conflict when I try to merge this into my branch
❓ Surprised that there's merge conflicts given I based it off of your branch 😕

> I don't think this is even needed, since I'm trying this locally and it seems to work fine without my changes?

Your commits are in this branch, so that would explain why everything you expect to work still works.

**Comment by rvantonder** [comment]
LGTM, PR it against master?

**Comment by rvantonder** [comment]
> ❓ Surprised that there's merge conflicts given I based it off of your branch

when I read: `wants to merge 1 commit into rvt/web-ui-structural-search from fa/accurate-toggle`, I assumed it is branched off of my commit. Let's change the base branch to master.

**Pull request closed without merging**